src/jsonrpccpp/common/specification.h

```diff
@@ -36,8 +36,9 @@ namespace jsonrpc
         JSON_BOOLEAN = 2,
         JSON_INTEGER = 3,
         JSON_REAL = 4,
-        JSON_OBJECT = 5,
-        JSON_ARRAY = 6
+        JSON_NUMERIC = 5,
+        JSON_OBJECT = 6,
+        JSON_ARRAY = 7
```

Comment:
Please don't change the ordinals of existing types. Add `JSON_NUMERIC=7`.

Reply:
Review comments addressed and added JSON_NUMERIC=7.
Kindly check the updated diff.